test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/PatchTests.cs

```diff
@@ -25,11 +28,76 @@ public PatchTests(HttpIntegrationTestFixture fixture)
 
         [Fact]
         [Trait(Traits.Priority, Priority.One)]
-        public async Task GivenAServerThatDoesNotSupportIt_WhenSubmittingAPatch_ThenMethodNotAllowedIsReturned()
+        public async Task GivenAServerThatSupportsIt_WhenSubmittingAPatch_ThenServerShouldPatchCorrectly()
```

Comment:
Add `[HttpIntegrationFixtureArgumentSets(DataStore.All, Format.Json)]` to the class to only run for Json